src/Sylius/Behat/Resources/config/suites/ui/checkout/addressing.yml

```diff
@@ -0,0 +1,21 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_checkout_addressing:
```

Comment:
We can start with just `checkout` suite and eventually split it only when it will be required.